browser-tests/functional-tests/accessibility.spec.js

```diff
@@ -0,0 +1,32 @@
+const { test, expect } = require("@playwright/test");
+const { iteratePagesAndContexts } = require("../data/pagesAndContexts.js");
+const AxeBuilder = require("@axe-core/playwright");
+
+test.describe.parallel("Accessibility tests", () => {
+  test.setTimeout(120000);
+  iteratePagesAndContexts(
+    (pageName, context, language, url) => {
+      test(`Accessibility check for ${pageName}, context ${context} and language ${language}`, async ({ page }) => {
+        await page.goto(url);
```

Comment:
Some of our pages have a details component that we should probably open before running the scan if we can. See https://playwright.dev/docs/accessibility-testing#configuring-axe-to-scan-a-specific-part-of-a-page
(is it possible to just try opening a details component if it's there?)

Reply:
Fine to leave it if it's a faff